utils/debufr.c.in

```diff
@@ -177,11 +178,12 @@ int main( int argc, char *argv[ ] ) {
 	char forcemt = 'N';
 	char cfms = 'Y';
 	char io = 'r';
-	char tbldir[MXFLEN] = "@MASTER_TABLE_DIR@";
+	char tbldir[MXFLEN] =
+		"@MASTER_TABLE_DIR_C@";
 	char outfile[MXFLEN];
 	char wkstr[MXFLEN];
 	char wkstr2[MXFLEN];
-	char tblfil[(MXFLEN*2)+5];
+	char tblfil[MXFLEN+30];
```

Comment:
What is the "30"?

Reply:
Further down in the code, tblfil is generated by appending a directory separator character ("/") and a BUFR tables filename to tbldir.  So tblfil needs to be dimensioned a bit bigger than tbldir, and no BUFR tables filename in this context should be longer than 26 characters, so I just rounded up to a nice even number ;-)

Reply:
Could you instead define another preprocessor symbol, like MAX_FILENAME_LEN or something suitable?

There should be no bare constants in production code. (Bare constants are acceptable and very useful in test code).

Reply:
OK, I just pushed up another commit which uses a new symbol MXFLEN_TBLFIL, and I added in a little extra wiggle room as well (50 characters instead of 30)

Reply:
If this looks OK to you now, would you mind going in and approving this PR so I can get it merged? (Thanks!)